data/courses.yaml

```
@@ -17,6 +17,17 @@
 # a general subject (`mathematics`, `logic`, `computer science`).
 #
 # This list is not ordered. Add new entries wherever you want.
+- name: Formalization of Mathematics: Seminar on the proof assistant Lean
```

The YAML parser will probably complain unless there are quotes here:
```suggestion
- name: "Formalization of Mathematics: Seminar on the proof assistant Lean"
```